packages/preload/src/eventsBridge.ts

```diff
@@ -91,3 +91,7 @@ export async function electronGoogleOAuth2Connector(): Promise<Credentials> {
 export async function createSpreadsheet(spreadsheetId: string, credentials: Credentials): Promise<string> {
   return electron.ipcRenderer.invoke('createSpreadsheet', spreadsheetId, credentials);
 }
+
+export async function sendUserInput(input: string) {
+  await electron.ipcRenderer.send('get-otp-response', input);
```

Comment:
Why `send` and not `invoke`?

Reply:
I'm still learning. What's the difference?

Reply:
I'm not sure there is a difference,

https://stackoverflow.com/a/59889863/839513

But all other calls are using `invoke`, so you should as well.

Reply:
Done